src/main/target/RG_SSD_F3/target.c

```diff
@@ -0,0 +1,23 @@
+#include <stdint.h>
+
+#include <platform.h>
+#include "drivers/io.h"
+
+#include "drivers/timer.h"
+#include "drivers/timer_def.h"
+#include "drivers/dma.h"
+
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
+	DEF_TIM( TIM2, CH4, PA3, TIM_USE_PWM | TIM_USE_PPM, TIMER_INPUT_ENABLED ) ,
```

Comment:
Oh, yeah Right. It's legacy code. Back in a day, there was 4 inputs. I'll clean this up.